com.github.Johnn3y.Forklift.json

```diff
@@ -0,0 +1,93 @@
+{
+  "id" : "com.github.Johnn3y.Forklift",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.34",
+  "sdk" : "org.gnome.Sdk",
+  "finish-args" : [
+    "--socket=wayland",
+    "--socket=x11",
+    "--share=ipc",
+    "--share=network",
+    "--filesystem=home"
+  ],
+  "cleanup" : [
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules" : [
+    "shared-modules/lame/lame-3.99.5.json",
+    {
+      "name" : "ffmpeg",
+      "config-opts" : [
+        "--enable-shared",
+        "--disable-static",
+        "--disable-doc",
+        "--disable-ffplay",
+        "--disable-devices",
+        "--enable-gnutls",
+        "--enable-libmp3lame",
+        "--enable-libvorbis"
+      ],
+      "sources" : [
+        {
+          "url" : "https://ffmpeg.org/releases/ffmpeg-4.0.2.tar.xz",
+          "sha256" : "a95c0cc9eb990e94031d2183f2e6e444cc61c99f6f182d1575c433d62afb2f97",
+          "type" : "archive"
+        }
+      ],
+      "cleanup" : [
+        "/share/ffmpeg"
+      ],
+      "post-install" : [
+        "install -Dm644 COPYING.LGPLv3 /app/share/licenses/ffmpeg/COPYING"
+      ]
+    },
+    {
+      "name" : "youtube-dl",
+      "buildsystem" : "simple",
+      "ensure-writable" : [
+        "/lib/python*/site-packages/easy-install.pth"
+      ],
+      "sources" : [
+        {
+          "url" : "https://github.com/rg3/youtube-dl",
+          "tag" : "2019.09.28",
+          "commit" : "f4b865c6136547a5af814526b65c08c89dbe3a14",
+          "type" : "git"
+        }
+      ],
+      "build-commands" : [
+        "python3 setup.py build",
+        "python3 setup.py install --prefix=/app"
+      ]
+    },
+    {
+      "name" : "libhandy",
+      "buildsystem" : "meson",
+      "config-opts" : [
+        "-Dexamples=false",
+        "-Dtests=false"
+      ],
+      "sources" : [
+        {
+          "url" : "https://source.puri.sm/Librem5/libhandy",
+          "branch" : "v0.0.11",
+          "commit" : "d0b3abf7a520e80c2c2155fa3da83962d2b80221",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "Forklift",
+      "buildsystem" : "meson",
+      "sources" : [
+        {
+          "url" : "https://github.com/Johnn3y/Forklift",
```

Comment:
If it's a stable release, you should create a tag in your repository and then re-use it here. (Not a blocker, just a good practice.)